great_expectations/rule_based_profiler/parameter_builder/parameter_container.py

```diff
@@ -17,6 +29,39 @@
 FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY: str = "value"
 FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY: str = "details"
 
+attribute_name = Word(alphas, alphanums + "_.") + ZeroOrMore(
```

Comment:
@cdkini I added a docstring -- otherwise, I followed a pattern we have used everywhere else for `pyparsing`.  If you think that we could make it nicer (more encapsulated), happy to brainstorm and do as part of a separate work for the entire code base.  Thanks!

Reply:
That's fair I'm okay leaving this be for now